src/notation/internal/notationinteraction.cpp

```diff
@@ -7339,6 +7339,57 @@ void NotationInteraction::addMelisma()
     toLyrics->selectAll(toLyrics->cursor());
 }
 
+void NotationInteraction::addElision() {
+
+    if (!m_editData.element || !m_editData.element->isLyrics()) {
+        LOGW("addElision called outside of lyrics editing context.");
+        return;
+    }
+
+    // fetch the text‐edit data & cursor
+    auto edd = m_editData.getData(m_editData.element);
+    if (!edd) {
+        LOGE("addElision: Failed to get EditData for the element.");
+        return;
+    }
+    auto* ted = static_cast<TextEditData*>(edd.get());
+    if (!ted) {
+        LOGE("addElision: EditData is not TextEditData.");
+        return;
+    }
+
+    TextCursor* cursor = ted->cursor();
+    if (!cursor) {
+        LOGE("addElision: Failed to get TextCursor from TextEditData.");
+        return;
+    }
+
+    // build the elision symbol and set the engraving font
+    String elisionStr = score()->engravingFont()->toString(mu::engraving::SymId::lyricsElision);
+    cursor->format()->setFontFamily(u"ScoreText");
+
+    // insert as one undoable action
+    score()->undo(new InsertText(cursor, elisionStr), &m_editData);
+
+    // force immediate layout & redraw of the lyrics line
+    auto* lyrics = toLyrics(m_editData.element);
+    lyrics->triggerLayout();
+    lyrics->score()->setLayoutAll(false);
```

Comment:
I don't see a method `setLayoutAll` that takes a boolean as argument. And are you sure this call is necessary? I don't immediately see why it would, or where it comes from...